### PR TITLE
Senior ID cards get a custom job name

### PR DESCRIPTION
**Pull request by Axionyxx**
## About the PR
Makes it actually worth taking, as it isn't just a resprite in the loadout

**Changelog**
:cl:
- tweak: Senior PDA's now have an ID card with a custom job title

### CODE REVIEW

**Comment by Axionyxx** [comment]
Gotta make some more changes so it syncs with station records

**Comment by Axionyxx** [comment]
image
image
Works as intended now

**Comment by WarMechanic** [comment]
This is a godsend for me as I enjoy teaching others engineering. Looking forward to seeing it implemented.

**Comment by redmushie** [comment]
Have you not read anything I wrote?

This PR is a bandaid solution ("rename one ID by hand") on top of a bandaid solution ("senior PDAs"). Virtually nobody uses the PDA box for anything, and this PR will not change that. It's still easier to use the ID card computer to achieve what this PR implements.

The end result is you've cluttered up the code -- something you claim to want to prevent when you **closed my PR** -- with custom changes that solve none of the stated problems.

**Comment by ProphetCrow** [comment]
^ Its very telling when you flag criticism as abuse.

File: Content.Server/Access/Components/PresetIdCardComponent.cs [hunk 1]
```diff
@@ -11,4 +11,10 @@ public sealed partial class PresetIdCardComponent : Component
 
     [DataField("name")]
     public string? IdName;
+
+    /// <summary>
+    /// DeltaV: Allow changing the job title, even if it'd be otherwise set by the JobPrototype
+    /// </summary>
+    [DataField("customJob")]
```
Comment (deltanedas) [inline Content.Server/Access/Components/PresetIdCardComponent.cs:18]: just [DataField]

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Content.Server/Access/Components/PresetIdCardComponent.cs (hunk 1 above)
Comment (Axionyxx) [reply Content.Server/Access/Components/PresetIdCardComponent.cs:18]: customJob vs customJobName, this is intended

**Review by deltanedas — APPROVED** [review]

**Comment by Axionyxx** [comment]
customJob just sounds better, I would've done just job, but it was already taken

**Pull request merged**